src/wyzeapy/wyze_auth_lib.py

```diff
@@ -233,11 +234,12 @@ async def refresh(self) -> None:
             response = await _session.post("https://api.wyzecam.com/app/user/refresh_token", headers=headers,
                                            json=payload)
         response_json = await response.json()
-        check_for_errors_standard(response_json)
+        check_for_errors_standard(self, response_json)
```

Comment:
This isn't the only place you'll need to modify. It looks like you would also have to perform this same check in the lock and hms variations of this check at least.